### PR TITLE
fix: Reset left nav capitalization

### LINKED ISSUE #156: Left Navbar Caps lock Titles
## Skills
Some of the skills you could see yourself learning & applying as you go along:

- Git/GitHub
- GoHugo
- HTML, CSS (& Bootstrap)

## What needs to be done

The side nav bar on [docs.atsign.com](docs.atsign.com) has full uppercased words. We believe it would look better if they weren’t. For example “GET STARTED” > “Get Started.”

## Support
In case you need any kind of help (such as getting your environment setup or asking questions about our tech) don’t be afraid to ask in our [discord](https://discord.atsign.com). There is a team full of developers ready to jump in and help on whatever your issue may be.

### PR DESCRIPTION
**Pull request by aswin2108**
**- What I did**
Fixed #156 
**- How I did it**
Added the `text-transform: capitalize` in _dark.scss file
image

**- How to verify it**
In the home page you can see the nav-bar to verify.
The ss:
image

**- Description for the changelog**
Added property of text-transform to change the caps-lock on the tiles.
Note: the change in hugo_stats.json file was not made by me, it got changed when I did `npm install`

### CODE REVIEW

**Review by TylerTrott — DISMISSED** [review]
I have tested your branch locally and these changes do not reflect on my side.

File: assets/scss/common/_dark.scss [hunk 1]
```diff
@@ -183,6 +183,7 @@ $navbar-dark-active-color:          $link-color-dark;
   color: $body-color-dark;
   background-color: transparent;
   border: 0;
+  text-transform: capitalize;
```
Comment (xavierchanth) [inline assets/scss/common/_dark.scss]: The _dark.scss file only applies to dark mode, please double check your work in light mode.

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Thanks for your work, however using capitalize poses an issue: the term `atPlatform` should be lower case 'a'. The goal here is to unset the `text-transform: uppercase` that is applied by another class. This can be done using `text-transform: none` to use the default text casing.

File: assets/scss/common/_dark.scss (hunk 1 above)
Comment (aswin2108) [reply assets/scss/common/_dark.scss]: Thanks! Will check it now itself!

**Comment by aswin2108** [comment]
@TylerTrott @XavierChanth Is this fix correct?
image

image

**Review by xavierchanth — APPROVED** [review]
Perfect, thanks!

**Comment by JeremyTubongbanua** [comment]
@aswin2108 
Hey! Thank you for your contribution. Your PR was accepted, merged, and it closed an issue. You could be eligible for a prize. 

To be eligible, please send an email to [hacktoberfest@atsign.com](mailto:hacktoberfest@atsign.com) with a screenshot of the PR approval email (or just forward the PR approval email to us). This is to prove that your email corresponds to the correct GitHub account. Thank you!

When it comes time to distribute the prizes, we will look at all the emails we've received from that email so don't forget to do this!

**Pull request merged**